java/ql/src/semmle/code/java/frameworks/Jackson.qll

```diff
@@ -77,14 +77,6 @@ class SetPolymorphicTypeValidatorSource extends DataFlow::ExprNode {
   }
 }
 
-/** Holds if `fromNode` to `toNode` is a dataflow step that resolves a class. */
-predicate resolveClassStep(DataFlow::Node fromNode, DataFlow::Node toNode) {
-  exists(ReflectiveClassIdentifierMethodAccess ma |
```

Comment:
Did this removal make any imports superfluous in this file?

Reply:
Yes, removed one